modules/react/modal/stories/examples/BodyOverflow.tsx

```diff
@@ -1,52 +1,52 @@
 import React from 'react';
 
 import {Modal} from '@workday/canvas-kit-react/modal';
-import {DeleteButton} from '@workday/canvas-kit-react/button';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
 import {HStack} from '@workday/canvas-kit-react/layout';
 
 export const BodyOverflow = () => {
-  const handleDelete = () => {
-    console.log('Deleted item');
+  const handlePrimaryClick = () => {
+    console.log('Primary Action clicked');
+  };
+
+  const handleSecondaryClick = () => {
+    console.log('Secondary Action clicked');
   };
 
   return (
     <Modal>
-      <Modal.Target as={DeleteButton}>Delete Item</Modal.Target>
+      <Modal.Target as={PrimaryButton}>Open License</Modal.Target>
       <Modal.Overlay>
         <Modal.Card>
           <Modal.CloseIcon aria-label="Close" />
-          <Modal.Heading>Delete Item</Modal.Heading>
+          <Modal.Heading>MIT License</Modal.Heading>
           <Modal.Body>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
+            <p>
+              Permission is hereby granted, free of charge, to any person obtaining a copy of this
+              software and associated documentation files (the "Software"), to deal in the Software
```

Comment:
Same feedback here as @mannycarrera4 had for the [Full Overflow](https://github.com/Workday/canvas-kit/pull/1621#issuecomment-1256391240) example: can we add more body text here to guarantee (more or less) the overflow is triggered for folks who view the example? Maybe we need to show a longer license 😄